### PR TITLE
[MIG] product_uom_updatable: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**

### CODE REVIEW

File: product_uom_updatable/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        for key, products_group in groupby(sorted_items, key=lambda r: r[field_name]):
+            product_ids = [product.id for product in products_group]
+            new_uom = uom_obj.browse(uom_id)
+            if (
+                key.category_id == new_uom.category_id
+                and key.factor_inv == new_uom.factor_inv
+            ):
+                # pylint: disable=sql-injection
+                query = sql.SQL(
+                    "UPDATE product_template SET {field} " " = %s WHERE id in %s"
```
Comment (petrus-v) [inline product_uom_updatable/models/product_template.py]: I'm not sure to understing quotting here, is it a linter resimt ?
 ```suggestion
                     "UPDATE product_template SET {field} = %s WHERE id in %s"
```

File: product_uom_updatable/models/product_template.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                ).format(field=sql.Identifier(field_name))
+                self.env.cr.execute(query, (new_uom.id, tuple(product_ids)))
+                self.invalidate_cache(fnames=[field_name], ids=product_ids)
+            else:
+                raise UserError(
```
Comment (petrus-v) [inline product_uom_updatable/models/product_template.py:44]: I've the feeling that `ValidationError` would be appropriate but won't require change for this !

**Review by petrus-v — APPROVED** [review]
Few details but LGTM reading the code

File: product_uom_updatable/models/product_template.py (hunk 2 above)
Comment (dzungtran89) [reply product_uom_updatable/models/product_template.py:44]: Thanks for your review @petrus-v

**Review by dsolanki-initos — APPROVED** [review]
Code and functional review. Looks good to me

**Comment by dsolanki-initos** [comment]
It would be great if someone can merge this PR.

**Comment by dsolanki-initos** [comment]
Can this be merged?

**Comment by dsolanki-initos** [comment]
@OCA/product-maintainers It would be great if this PR is merged, hence  it is approved.

**Comment by dsolanki-initos** [comment]
It would be nice if maintainers could merge the PR.
Thank you in advance!

**Comment by dsolanki-initos** [comment]
It would be nice if maintainers could merge the PR.
Thank you in advance!

**Comment by dsolanki-initos** [comment]
It would be nice if the  maintainers can merge the PR

**Comment by dsolanki-initos** [comment]
It would be nice if the maintainers can merge the PR

**Comment by dsolanki-initos** [comment]
It seems that PR is approved it would be nice if maintaienr merge it.

**Comment by dsolanki-initos** [comment]
It seems that PR is approved it would be nice if maintainer can merge it.

**Comment by dsolanki-initos** [comment]
It seems that PR is approved it would be nice if maintainer can merge it.

**Pull request merged**